ui/admin/app/routes/scopes/scope/auth-methods/auth-method/managed-groups.js

```diff
@@ -19,16 +19,24 @@ export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsRoute extends
    *
    * @returns {Promise{[ManagedGroupsModel]}}
    */
-  model() {
+  async model() {
     const authMethod = this.modelFor('scopes.scope.auth-methods.auth-method');
     const { id: auth_method_id } = authMethod;
     const canListManagedGroups = this.can.can('list collection', authMethod, {
       collection: 'managed-groups',
     });
+    let managedGroups;
 
     if (canListManagedGroups) {
-      return this.store.query('managed-group', { auth_method_id });
+      managedGroups = await this.store.query('managed-group', {
+        auth_method_id,
+      });
```

Comment:
Yes.  But the type of errors that can occur are well known.  We don't expect unknown JavaScript errors to occur here.  However, we could handle these more gracefully in the future, just in case.  I'd say though this is an edge case that falls outside the scope of this PR.

Reply:
Yeah as long as HTTP errors are fine then this is good in my book, I half guessed you'd have this covered anyhow, it was mainly just checking/chatting, so all good. No urgency at all on catch just plain old JS errors, as you said they are in all likelihood gonna be caught by you folks not our users, so if it was me I wouldn't worry about that too much.